greet.spec.js

```diff
@@ -0,0 +1,8 @@
+import { greet } from "./greet";
+
+describe("test for the greet function", () => {
+  it("if the name equal Thomas or Martin or Stefan return 'Hello Coach'", () => {
+    const result = greet("Thomas");
+    expect(result).toBe("Hello Coach!");
```

Comment:
Very good test.
To be confident the most, I recommend adding two more tests at least. To test if the result is the same when providing `Martin` and `Stefan`. Right now, you just test `Thomas`

Reply:
done